src/main/java/tyrone/constants/Constants.java

```diff
@@ -0,0 +1,133 @@
+package tyrone.constants;
+
+import java.util.ArrayList;
+import tyrone.task.Deadline;
+import tyrone.task.Event;
+import tyrone.task.Task;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Constants {
+    public static final String LINE = "    ___________________________________";
+    public static String logo = " _____                           \n|_   _|   _ _ __ ___  _ __   ___ \n  | || | | | '__/ _ \\| '_ \\ / _ \\\n  | || |_| | | | (_) | | | |  __/\n  |_| \\__, |_|  \\___/|_| |_|\\___|\n      |___/                      ";
+
+    public static ArrayList<Task> toDoList = new ArrayList<>();
+
+    public Constants() {
+    }
+
+    public static void goodbye() {
+        System.out.println(LINE);
+        System.out.println("    see you brother");
+        System.out.println(LINE);
+    }
+
+    public static void markAsDone(int index) {
+        toDoList.get(index).setDone(true);
+        System.out.println(LINE);
+        System.out.println("    TIGHT! I've marked this task as done:");
+        System.out.println("      [X] " + toDoList.get(index).getDescription());
+        System.out.println(LINE);

```

Comment:
You can consider extracting this part into a method and pass in parameters to differentiate with ```unmarkAsUndone``` as they are pretty similar.